src/main/scala/vectorpipe/osm/internal/PlanetHistory.scala

```diff
@@ -61,12 +70,18 @@ private[vectorpipe] object PlanetHistory {
         .distinct
```

Comment:
This is a little out of context but why is `.distinct` required here? Is it that a way can have multiple references to a single node or that there can somehow be two way records with same ID that may potentially reference the same node ?

Reply:
> ...be two way records with same ID that may potentially reference the same node

Precisely, the `.distinct` has to remain or the function will return an incorrect result.